### PR TITLE
MMDLoader: Skip transparency check for CompressedTexture.

### PR DESCRIPTION
**Pull request by bill42362**
Related issue: #17974 

**Description**

I got this similar error when load MMD model with .dds texture:

```
MMDLoader.js?aed8:1456 Uncaught TypeError: Failed to execute 'drawImage' on 'CanvasRenderingContext2D': The provided value is not of type '(CSSImageValue or HTMLImageElement or SVGImageElement or HTMLVideoElement or HTMLCanvasElement or ImageBitmap or OffscreenCanvas or VideoFrame)'
    at createImageData (MMDLoader.js?aed8:1456)
    at Array.eval (MMDLoader.js?aed8:1524)
    at eval (MMDLoader.js?aed8:1406)
    at Object.eval [as onLoad] (three.module.js?5a89:35025)
    at XMLHttpRequest.eval (three.module.js?5a89:34742)
```

And I found if there's data property inside image object, [MMDLoader:L1316](https://github.com/mrdoob/three.js/blob/dev/examples/js/loaders/MMDLoader.js#L1316) would just use it to check transparent, which could bypass process to get image data via `<canvas>`.

First time to make PR to opensource projects, hope this could help.

### CODE REVIEW

**Comment by Mugen87** [comment]
I think this problems should be solved in `MMDLoader` instead.

I also doubt that `detectImageTransparency()` will work with a compressed texture.

**Comment by bill42362** [comment]
@Mugen87 Thanks for your comment!

I've change the fix into MMDLoader, but I don't know better ways to check transparency of compressed textures.
So it's just a quick fix, which only set transparent to `false` for non-alpha channel formats.

**Review by takahirox — APPROVED** [review]
Yes, as @Mugen87 mentioned I don't think `detectImageTransparency()` works for compressed texture. So this workaround looks good to me (for now). Thanks!

File: examples/jsm/loaders/MMDLoader.js [hunk 1]
```diff
@@ -1519,6 +1533,23 @@ class MaterialBuilder {
 
 			}
 
+			if ( texture instanceof CompressedTexture ) {
```
Comment (mrdoob) [inline examples/jsm/loaders/MMDLoader.js:1536]: It'd be better to do this check instead:

```js
if ( texture.isCompressedTexture === true ) {
```

Comment (bill42362) [reply examples/jsm/loaders/MMDLoader.js:1536]: Hi, @mrdoob 
should I just push on this branch or make another PR?

Comment (mrdoob) [reply examples/jsm/loaders/MMDLoader.js:1536]: You can just push to this branch 👍

Comment (bill42362) [reply examples/jsm/loaders/MMDLoader.js]: updated, plz take a look, thx!

**Comment by mrdoob** [comment]
Thanks!

**Pull request merged**